containers/src/Kokkos_OffsetView.hpp

```diff
@@ -1094,6 +1094,43 @@ class OffsetView : public ViewTraits<DataType, Properties...> {
     return m_track.template get_label<typename traits::memory_space>();
   }
 
+
+  private:
+  template <class T, int R,
+            bool P = (R > 1) && (std::is_same<T, Kokkos::LayoutLeft>::value ||
+                                 std::is_same<T, Kokkos::LayoutRight>::value)>
+  struct is_strided_from_padding;
```

Comment:
Rather than repeat this helper class as a private member of each `View`-like class, could it just go somewhere in `Kokkos::Impl`?

Reply:
Yes.